doc/source/install.rst

```diff
@@ -1,59 +1,100 @@
+``````````
 
-Conda-based distributions (Anaconda, Miniconda)
------------------------------------------------
+On Windows, you can install ``scikit-image`` using::
 
-From the shell/command prompt, execute::
+    pip install scikit-image
 
-  conda install scikit-image
+For Conda-based distributions (Anaconda, Miniconda), execute::
 
-Windows
--------
-``scikit-image`` comes pre-installed with several Python
-distributions, including Anaconda_, `Enthought Canopy`_,
-`Python(x,y)`_ and `WinPython`.
+    conda install scikit-image
+
+If you are using pure Python i.e. the distribution from python.org_, you'll need to
+manually download packages (such as numpy, scipy and scikit-image) using Python wheels available from
+`Christoph Gohlke's`_ website. You can install Python wheels using::
+
+    pip install SomePackage-1.0-py2.py3-none-any.whl
 
 .. _Anaconda: https://store.continuum.io/cshop/anaconda/
 .. _Enthought Canopy: https://www.enthought.com/products/canopy/
 .. _Python(x,y): http://code.google.com/p/pythonxy/wiki/Welcome
 .. _WinPython: https://winpython.github.io/
 
-If you are using the distribution from python.org_, you'll need to
-manually download a few packages: numpy, scipy and scikit-image from
-`Christoph Gohlke's`_ website. Python wheels are installed using::
-
-  pip install SomePackage-1.0-py2.py3-none-any.whl
+b. Debian and Ubuntu
+````````````````````
 
-Debian and Ubuntu
------------------
-On Debian and Ubuntu install scikit-image with::
+On Debian and Ubuntu, install ``scikit-image`` with::
 
   sudo apt-get install python-skimage
 
-Building from source
---------------------
+2. Development Installation:
+----------------------------
+
+You can install ``scikit-image`` development version if either your distribution ships an outdated version
+or you want to develop and work on new features before the package is released officially.
+
+a. Windows
+``````````
+
+Before installing the development version, uninstall the standard version of ``scikit-image`` using pip as::
+
+  pip uninstall scikit-image
+
+or using conda (for Anaconda users) as::
+
+  conda uninstall scikit-image
 
-Execute the following command from the shell::
+Now clone scikit-image on your local computer::
 
-  pip install scikit-image
+  git clone https://github.com/scikit-image/scikit-image.git
+
+Change the directory and build from source code::
+
+  cd scikit-image
+  python setup.py develop
 
-Windows
-```````
 If you experience the error ``Error:unable to find vcvarsall.bat`` it means that
-distutils is not correctly configured to use the C compiler. Modify (or create,
-if not existing) the configuration file ``distutils.cfg`` (located for
-example at ``C:\Python26\Lib\distutils\distutils.cfg``) to contain::
+your computer does not have recommended compilers for python. You can download and
+install Windows compilers from `here`_  or correctly configure distutils
+to use the C compiler by modifying (or create, if not existing) the configuration file
```

Comment:
These are two different types of compilers.  The link you added is to Microsoft's compiler, while MinGW is a community project, and the original instructions had implicitly assumed the developer was using it.   I would say something like "install Windows compilers from <here> or, if using the MinGW compiler <link>, configure distutils to use it by modifying..."